contributing/code/standards.rst

```diff
@@ -167,6 +167,29 @@ Service Naming Conventions
 
 * A group name uses the underscore notation.
 
+.. _contributing-code-standards-comparisons:
+
+Comparisons
+-----------
+
+Use `identical comparison`_ when the expected value must match a specific type:
```

Comment:
I would go further and recommend to always use the identical comparison unless you have a strong reason to not do that. @WouterJ @xabbuh what do you think?